rust-libretro/src/environment.rs

```diff
@@ -939,14 +1005,13 @@ pub unsafe fn set_hw_render_context_negotiation_interface(
 pub unsafe fn set_serialization_quirks(
     callback: retro_environment_t,
     quirks: SerializationQuirks,
-) -> bool {
+) -> Result<(), EnvironmentCallError> {
```

Comment:
`RETRO_ENVIRONMENT_SET_SERIALIZATION_QUIRKS` returns a modified set of quirks flags, because the frontend is expected to clear any bits that it doesn't recognize.

Reply:
`SerializationQuirks` should also be based on `u64`, I’ll fix this as well.